ui/src/stylesheets/toc.scss

```
@@ -43,6 +43,19 @@
   padding-right: var(--sm);
   overflow: hidden;
   text-overflow: ellipsis;
+  font-family: var(--sans-serif);
+  font-weight: var(--weight-bold);
+  font-size: var(--md);
+  opacity: .8;
+
+  &:hover {
+    color: var(--core-violet-2);
+  }
+
+  &.subheading-list {
```

I don't think this rule is doing what you want. I'm seeing all of the ToC items in bold.

Try this:

```suggestion
  &.subheading-item {
```